packages/dds/tree/src/test/shared-tree/fuzzEditGenerator.ts

```diff
@@ -42,6 +42,7 @@ export interface FuzzInsert {
 export interface FuzzDelete {
 	fuzzType: "delete";
 	path: UpPath;
+	nodesToDelete: number;
```

Comment:
Instead of adding these fields, it would be nice to just extend `PathWithNodeRange`.